q2_health_index/plugin_setup.py

```diff
@@ -18,12 +19,18 @@
     short_description='Plugin for calculating the Gut Microbiome Health Index (GMHI).'
 )
 
-plugin.visualizers.register_function(
-    function=print_hello,
-    inputs={},
-    parameters={},
-    input_descriptions={},
-    parameter_descriptions={},
-    name='Print hello.',
-    description='Print hello on the screen.'
+plugin.methods.register_function(
+    function=calculate_gmhi,
+    inputs={'table': FeatureTable[RelativeFrequency],
```

Comment:
`inputs={'table': FeatureTable[Frequency]` or am I missing something and it should be `RelativeFrequency`?

Reply:
In the paper they've used relative abundances, therefore we implemented data import through `FeatureTable[RelativeFrequency]`. This is already tested and working and will be added in the next commit.

Reply:
so just to be clear, `FeatureTable[RelativeFrequency]` has relative frequencies, ie. not counts. If we want to use raw Feature Tables at some point, they come as `FeatureTable[Frequency]` as they would need to be transformed before using.

Reply:
This makes sense, as they discard unclassified species, and then normalize the table. Good point, already corrected!

Reply:
so just to be clear - that implies that the plugin would have another method that takes raw feature table and transforms it to relative frequency table. One way to go about it then, would be to have 2 methods and 1 pipeline:
methods:
* transform `FeatureTable[Frequency]` to `FeatureTable[RelativeFrequency]`
* calculate the GMHI on `FeatureTable[RelativeFrequency]`
plpeline:
* do both from the above